emailer.py

```diff
@@ -38,21 +38,6 @@ def init_rollbar():
         rollbar.contrib.flask.report_exception, app)
 
 
-@app.before_request
-def app_before_request():
-    envelopes.connstack.push_connection(
-        envelopes.SendGridSMTP(
-            login=os.environ.get('SENDGRID_USERNAME'),
```

Comment:
Probably some documentation changes needed to the top-level README now (e.g. `SENDGRID_PASSWORD` -> `MAILGUN_PASSWORD`)